### ISSUE TITLE
`ansible-vault: command not found`, perhaps unique to when running in Python venv

### ISSUE THREAD

**Issue by eengstrom** [ansible/vscode-ansible#701]
### Summary

TLDR: `ansible-vault: command not found`

Details:

When editing a file that is encrypted with `ansible-vault` (e.g. a `group_vars/GROUP/vault.yml`, I try to issue the command **"Ansible Vault: encyrpt/decrypt via `ansible-vault`"**, but get in return a `/bin/sh: ansible-vault: command not found`.

Yes, the language on the yaml file that I'm trying to decrypt is indeed `Ansible`.

Note that I run my Ansible out of a Python venv, but that works fine for other tasks like `ansible-lint`. I've looked at the code base for this extension, but cannot find where calls to `ansible-lint` are passing through the `venv` and perhaps similar calls to `ansible-vault` are not.

### Extension version

v1.0.90

### VS Code version

1.73.1

### Ansible Version

```shell
ansible 2.10.17
  config file = $HOME/Projects/sysops/ansible/ansible.cfg
  configured module search path = ['$HOME/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = $HOME/Projects/sysops/ansible/.venv/lib/python3.8/site-packages/ansible
  executable location = $HOME/Projects/sysops/ansible/.venv/bin/ansible
  python version = 3.8.9 (default, Apr 13 2022, 08:48:06) [Clang 13.1.6 (clang-1316.0.21.2.5)]
```

### OS / Environment

macOS Monterey (12.6.1), MBP M1 Max

```
% ansible-vault --version
ansible-vault 2.10.17
  config file = $HOME/Projects/sysops/ansible/ansible.cfg
  configured module search path = ['$HOME/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = $HOME/Projects/sysops/ansible/.venv/lib/python3.8/site-packages/ansible
  executable location = $HOME/Projects/sysops/ansible/.venv/bin/ansible-vault
  python version = 3.8.9 (default, Apr 13 2022, 08:48:06) [Clang 13.1.6 (clang-1316.0.21.2.5)]
```

### Relevant log output

```shell
Decryption of $HOME/Projects/sysops/ansible/playbooks/group_vars/gitlab/vault.yml failed: Error: Command failed: ansible-vault decrypt "/Users/engstrom/Projects/sysops/ansible/playbooks/group_vars/gitlab/vault.yml"
/bin/sh: ansible-vault: command not found
```

**Comment by eengstrom** [comment]
I made my best stab at looking at the code base for the extension, thinking that the issue is that calls to `ansible-vault` are not wrapped or called within my local Python venv, even though it appears as if calls to `ansible-lint` are.  

However, I'm not yet VSCode Extension "enabled", though eager to learn.  If someone wants to point me in the right direction for how calls to `ansible-lint` are successful when using a local venv, I'm happy to try to fix myself.

**Comment by priyamsahoo** [comment]
If using virtual environments, try adding this setting, and it should work:
```
ansible.python.activationScript: path/to/venv/activate-script
```
This will be used to run commands in a Python virtual environment

**Comment by eengstrom** [comment]
I'm sorry, I delayed in responding, but @priyamsahoo - I tried your suggestion to set `ansible.python.activationScript`, but that is never even called.  I substituted a test script that would log to a temporary file just to test. 

I was able to make a workaround by developing a full script called `ansible-vault` which runs the command inside the (activated) environment, but it's a terrible kludge.  

In other words, I don't think the setting is being respected for calls to `ansible-vault`, AFAICT.  

Any other suggestions?

**Comment by pgalbraith** [comment]
This is not working for me, either, I don't think this should have been closed.
